leaf-ui/js/displayAnalysis.js

```diff
@@ -270,8 +273,8 @@ function updateHistory(currentAnalysis){
 function addFirstAnalysisConfig(){
     $(".config-elements").css("background-color", "");
     $(".result-elements").css("background-color", "");
-    var id = "Configuration1"
-    currAnalysisConfig = new AnalysisConfiguration(id, analysisRequest);
+    var id = "Configuration1";
```

Comment:
you should probably use that highestPosition var here just to be consistent

Reply:
also if you implement the "readd a default config when all are deleted", you could use this func and just add a line that resets highestPosition to 1 in this func